neo/rawio/examplerawio.py

```diff
@@ -111,12 +112,18 @@ def _parse_header(self):
         # In short `_parse_header()` can be slow but
         # `_get_analogsignal_chunk()` needs to be as fast as possible
 
-        # create fake signal streams information
+        # create fake signal streams and buffer information
+        # a buffer is a group of channels that are in the same buffer for instance hdf5 or binary : this is optional
```

Comment:
I'm wondering if we want to weird this differently. buffer is not optional in the developer sense because it has to be a value or `""` but is optional in the sense that you can give an `""` string. So maybe we make that clearer. Let me think about this while you fix the tests.